db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/StateMachines/Methods.class/__methods__/catalogiteminitialization.rb

```diff
@@ -156,6 +156,17 @@ def set_option_on_provision_task(dialog_options_hash, prov)
   end
 end
 
+def set_option_on_destination(dialog_options_hash, destination)
+  dialog_options_hash.each do |key, value|
+    log_and_update_message(:info, "Adding Option: {#{key} => #{value}} to Destination id: #{destination.id}")
+    destination.set_dialog_option(destination_key_name(key), value)
+  end
+end
+
+def destination_key_name(key)
+  key.to_s.starts_with?("dialog_") ? key.to_s : "dialog_#{key}"
+end
```

Comment:
No, the field gets set in the destination (in this case service object) the same as it would if it was just a catalog item being ordered directly.  But it would be useful if @bzwei could validate this as well.